conf/broker.conf

```diff
@@ -344,14 +332,17 @@ loadBalancerNamespaceBundleMaxTopics=1000
 loadBalancerNamespaceBundleMaxSessions=1000
 
 # maximum msgRate (in + out) in a bundle, otherwise bundle split will be triggered
-loadBalancerNamespaceBundleMaxMsgRate=1000
+loadBalancerNamespaceBundleMaxMsgRate=5000
```

Comment:
If it's number of messages (and not BK entries), we could raise the threshold here (10K, 30K, 50K ? )

In any case, this is only one dimension, for example if the bytes/s are > 100MB we're going to split anyway.

Reply:
yes, I was also thinking to increase to higher value. I will change it to 30K.